packages/terra-heading/src/Heading.jsx

```diff
@@ -84,14 +79,10 @@ const Heading = ({
     attributes.className,
   ]);
 
-  const headingStyles = {
-    color,
-  };
-
   const Element = `h${level}`;
 
   return (
-    <Element {...attributes} style={{ ...headingStyles, ...customProps.style }} className={TextClassNames}>
+    <Element {...attributes} style={{ ...customProps.style }} className={TextClassNames}>
```

Comment:
Can remove this customProps.style spread now that we aren't combining styles.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2211/commits/35775cd63f5cf0ae778b8946046ff2e9a349eea6.